package.json

```diff
@@ -25,15 +26,18 @@
   "author": "devsupport@webex.com",
   "license": "MIT",
   "dependencies": {
-    "@webex/components": "^1.18.1",
+    "@webex/component-adapter-interfaces": "^1.2.1",
+    "dotenv": "^8.2.0",
     "rxjs": "^6.5.3",
     "webex": "^1.79.0"
```

Comment:
We should make `webex` a peer dependency

Reply:
this can not be a peer dependency in this repo. we are utilizing it in our scripts now.

Reply:
I will change it a `devDep`

Reply:
I'm ok with that, we don't want to bundle our implementation with the SDK.